test/queues.c

```diff
@@ -190,6 +190,25 @@ TEST test_queue_notification_close_histignore(void)
         PASS();
 }
 
+TEST test_queue_notification_skip_display(void)
+{
+        struct notification *n;
+
+        // Test skipping display
+        n = test_notification("n", -1);
+        n->skip_display = true;
+
+        queues_init();
+        queues_notification_insert(n);
+        QUEUE_LEN_ALL(0, 0, 1);
```

Comment:
Honestly, I'm missing another check for the signal.

Reply:
How do we check for the signal?  Note that this code is exercising the queues, not end-to-end dbus functionality (the signal is done in `dbus.c` not `queues.c`).  I don't see any end-to-end tests that also involve rules.